C/cp/missingElementsFasterMethod.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <math.h>
+
+int main()
+{
+
+    int arr[6];
```

Comment:
Why have you initialized the size of the array? You can take the size of the array as an input from the user.